app/main.py

```diff
@@ -1,8 +1,53 @@
+from decimal import Decimal
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: float) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        cost = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                cost += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return cost
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power <= car.clean_mark:
+            return 0
+
+        price = round(float(
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * Decimal(str(self.average_rating))
+            / Decimal(str(self.distance_from_city_center))
+        ), 1)
+
+        return price
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
```

Comment:
<img width="602" alt="image" src="https://github.com/mate-academy/py-car-wash-station/assets/93856423/4ddd836e-df73-4ae1-9070-81e274c2818b">

Reply:
![image](https://github.com/mate-academy/py-car-wash-station/assets/143685044/7dd8cb72-7b25-4d18-8fcd-f9990eb31c36)

CarWashStation should have such methods:

1. serve_cars - method, that takes a list of Car's, washes only cars with clean_mark < clean_power of wash station and returns income of CarWashStation for serving this list of Car's, rounded to 1 decimal: